projects/example_project/dummy/dummy_resnet.py

```diff
@@ -0,0 +1,16 @@
+from mmdet.models.backbones import ResNet as MMDET_RESNET
```

Comment:
Only global variables need to be capitalized
```suggestion
from mmdet.models.backbones import ResNet
```